content/applications/general/iot/config/updating_iot.rst

```diff
@@ -73,6 +73,81 @@ Lastly, click :guilabel:`Flash`, and wait for the process to finish.
    An alternative software for flashing the micro SD card is `Raspberry Pi Imager
    <https://www.raspberrypi.com/software/>`_.
 
+Windows IoT update
+==================
+
+Windows virtual :abbr:`IoT (Internet of Things)` box may occasionally need an update to work
+properly. The following processes are the un-installation and re-installation of the Windows virtual
+:abbr:`IoT (Internet of Things)` box.
+
+.. important::
+   Before un-installation of a previous version of the Windows virtual :abbr:`IoT (Internet of
+   Things)` box, check Odoo's latest download page to ensure a newer version
+   of Odoo's virtual :abbr:`IoT (Internet of Things)` box is available. To reach the download site
+   for the latest update use the following URL: `https://nightly.odoo.com/16.0/nightly/windows/`. It
+   is imperative that the `16.0` be replaced with the version of Odoo that the database is running
+   on (only full versions or long supported versions). The latest file will have `latest` in its
+   filename. Note the date of upload to the right of the filename.
+
+Uninstalling Windows IoT
+------------------------
+
+Prior to upgrading the Windows virtual :abbr:`IoT (Internet of Things)` box, the previous version
+should be uninstall first. Uninstalling the Windows virtual :abbr:`IoT (Internet of Things)` box is
+done through the Windows program manager. Using any Windows version, search for `program`. Then,
+from the control panel, select :guilabel:`Add or Remove Programs`. Then, search for `Odoo` and click
+the :guilabel:`.. (three dot)` (ellipsis) menu to uninstall.
+
+Confirm the un-installation and follow the steps to uninstall through the Odoo uninstall wizard.
+
+Download and re-installation
+----------------------------
+
+To begin the re-installation, navigate to the latest installation package for Enterprise or
+Community - Windows edition at  `Odoo's nightly download page
+<https://nightly.odoo.com/16.0/nightly/windows/>`_.
+
+.. important::
+   It is imperative that the `16.0` be replaced with the version of Odoo that the database is
+   running on. The latest file will have `latest` in its filename.
+
+Next, install and setup the Odoo :file:`.exe` file. After the instructions screen, click
+:guilabel:`Next` to start the installation and agree to the :abbr:`TOS (Terms of Service)`.
+
+During the next step of the re-installation, select :guilabel:`Odoo IoT` from the :guilabel:`Select
+the type of install` drop-down menu.
+
+.. example::
+   For reference, the following should be installed:
+
+   - **Odoo server**
+   - **Odoo IoT**
+   - **Nginx WebServer**
+   - **Ghostscript interpreter**
```

Comment:
Should these be bolded, as they currently are...or italicized, as we typically do with app/software/program names. What do you think?

Reply:
I've added gui labels instead. These are options on the screen